iXBRLViewerPlugin/__init__.py

```diff
@@ -199,7 +199,7 @@ def iXBRLViewerCommandLineXbrlRun(cntlr, options, *args, **kwargs):
     generateViewer(
         cntlr=cntlr,
         saveViewerDest=options.saveViewerDest or kwargs.get("responseZipStream"),
-        viewerURL=options.viewerURL,
+        viewerURL=options.viewerURL or DEFAULT_VIEWER_PATH,
```

Comment:
Every call to `generateViewer` now does some version of `viewerURL=X or DEFAULT_VIEWER_PATH`. It would be better to move this into the `generateViewer` function itself.